content/tutorials/build-apps/build-a-desktop-wallet-in-javascript.md

```diff
@@ -0,0 +1,1460 @@
+
+4. Update the view logic - in `view/template.html` add the following lines just before the `<input>` element with `id="input-destination-address`:
+
+```html
+<div class="input-group mb-3">
+    <!-- Step 8 code additions - start -->
+    <div class="accountVerificationIndicator">
+        <span>Verification status:</span>
+    </div>
+    <!-- Step 8 code additions - end -->
+    <input type="text" class="form-control" placeholder="rP4zcp52pa7ZjhjtU9LrnFcitBUadNW8Xz" id="input-destination-address">
+    <span class="input-group-text">To (Address)</span>
+</div>
+```
+
+5. Lastly, modify the renderer as described below:
+
+```javascript
+modalButton.addEventListener('click', () => {
+    modalDialog.show()
+})
+
+// Step 8 code additions - start
+const accountVerificationEl = document.querySelector('.accountVerificationIndicator span')
+// Step 8 code additions - end
+
+const destinationAddressEl = document.getElementById('input-destination-address')
+const destinationTagEl = document.getElementById('input-destination-tag')
+const amountEl = document.getElementById('input-xrp-amount')
+const sendXrpButtonEl = document.getElementById('send-xrp-submit-button')
+
+// Step 8 code additions - start
+destinationAddressEl.addEventListener('input', (event) => {
+    window.electronAPI.onDestinationAccountChange(destinationAddressEl.value)
+})
+
+window.electronAPI.onUpdateDomainVerificationData((_event, result) => {
+    accountVerificationEl.textContent = `Domain: ${result.domain || 'n/a'} Verified: ${result.verified}`
+})
+// Step 8 code additions - end
+
+sendXrpButtonEl.addEventListener('click', () => {
+    modalDialog.hide()
+    const destinationAddress = destinationAddressEl.value
+    const destinationTag = destinationTagEl.value
+    const amount = amountEl.value
+
+    window.electronAPI.onClickSendXrp({destinationAddress, destinationTag, amount})
+})
+```
+
+Run the following command:
+
+```console
+npm run start
+```
+
+Test your wallet app the same way you did in the previous steps. It should display a hint about the receiving account when opening up the "Send XRP" dialog and entering the address. To test domain verification, try entering the following addresses in the "To" box of the Send XRP dialog:
+
+| Address                              | Domain       | Verified? |
+|:-------------------------------------|:-------------|:----------|
+| `rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW` | `mduo13.com` | ✅ Yes    |
+| `rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn` | `xrpl.org`   | ❌ No     |
+| `rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe` | (Not set)    | ❌ No     |
+
+To run the reference application, run:
+
+```console
+npm run domain-verification
+```
+
+## Summary 
+
+Congratulations, you now have created your own wallet application! In completing this tutorial, you have learned how to interact with the XRP Ledger - from sending simple requests up to doing actual transactions, all in an electron application.
```

Comment:
I'd recommend expanding on this so it's at least 2 paragraphs. Otherwise the end seems too sudden considering how long each previous section was. It should call out each of the features you've added from every step so far. (Starting from Hello World we were able to add the ability to view the ledger, then you added...) - A numbered list calling back to each section would be good to remind them of all the cool stuff they did :)

I'd also recommend ending with a screenshot of the completed app like you showed at the beginning.

Reply:
Done, I'll leave it open for review